getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+    }
+   ],
+   "source": [
+    "from langchain_core.output_parsers import StrOutputParser\n",
+    "\n",
+    "chain = chat_prompt | model | StrOutputParser()\n",
+    "\n",
+    "print(\n",
+    "    chain.invoke(\n",
+    "        {\n",
+    "            \"country\": \"Japanese\", \n",
+    "            \"question\": \"What is the most popular Sashimi in Japan vs the rest of the world?\"\n",
+    "        }\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f186179-13e7-4056-9b62-9b4b1ac8de60",
+   "metadata": {},
+   "source": [
+    "### Wait in between API calls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f747f98-7b78-436f-a889-e3cff3f19bf9",
+   "metadata": {},
+   "source": [
+    "Depending on the users Azure plan and account will determine how many API requests can be made within a given interval. If too many requests are sent frequently an error may occur where the user will be told to wait **x** amount of time before sending another request.\n",
+    "\n",
+    "When creating a model, one of the parameters briefly mentioned earlier was the `max_retries` parameter. When calling the underlying Python OpenAI library, the library will wait and retry the call on your behalf at least 2 times (the default) before raising a RateLimiteError, or whatever `max retries` is set as.\n",
+    "\n",
+    "Otherwise, to ensure the user is not overwhelming the Azure services, make sure to wait between API calls when possible - this can be enforced by using the `time` package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 65,
+   "id": "99e45cb7-6e35-4420-bd41-651157e5596f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "time.sleep(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4a85fe6-ee18-4fc8-88c2-b67297ac8ef8",
+   "metadata": {},
+   "source": [
+    "### Check the costs and token usage of a given model API call"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "id": "561dfbfb-0933-4650-8230-23f1bc2b1f46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_community.callbacks.manager import get_openai_callback"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 68,
+   "id": "c9081591-0b58-44b3-89c7-c3a512f84f4c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "messages = [\n",
+    "    SystemMessage(content=\"Translate the following from German into English\"),\n",
+    "    HumanMessage(content=\"What's the first planet in the Solar System?\"),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61c9a128-f702-4f48-b695-436629e5ac1a",
+   "metadata": {},
+   "source": [
+    "`get_openai_callback()` is a context manager of the [**OpenAICallbackHandler**](https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/callbacks/openai_info.py) class, meaning it calls this class and creates an instance when used."
```

Comment:
I think it might be important to note, that with Azure OpenAI, you need to pass the model/model_version to get an accurate estimation of cost. The deployment name doesn't convey enough information to calculate this.

Reply:
I've included a short paragraph to explain this:

'Below is an example of how to use the `get_openai_callback()`. However, to get an accurate estimation of cost, you must pass the model and model version as parameters to the `AzureChatOpenAI` [**constructor**](#creating-an-azurechatopenai-model). '

Still not sure if my links to other sections in the document are working though :/